.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+inherit_from:
```

Comment:
Why add this here?  I don't think we need to rubocop anything in this repo, do we?

Reply:
This is for the scripts that live in this repo.

Reply:
👍